app/src/main/AndroidManifest.xml

```diff
@@ -22,7 +22,7 @@
         android:roundIcon="@drawable/logo"
         android:supportsRtl="true"
         android:theme="@style/AppTheme.Launcher"
-        android:windowSoftInputMode="adjustNothing">
+        android:windowSoftInputMode="adjustNothing|adjustPan">
```

Comment:
Why adding adjustPan? Maybe adjustResize would be better but is there a reason to add it?

Reply:
Yeah so for some reason the keyboard was moving up the buttons again. I just realized that the chat view is also moving up the toolbar and the message on top. I will try to fix it

Reply:
Yeah indeed I changed it to adjustNothing in the chatPage PR to make the messages show correctly while typing and didn't notice the buttons issue, my bad. Maybe there's a way to separate the behavior of the soft keyboard in different pages.